website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -62,6 +62,10 @@ default value in the `"/sys/config/ui"` [API endpoint](/vault/api-docs/system/co
   [go-sockaddr template](https://pkg.go.dev/github.com/hashicorp/go-sockaddr/template)
   that is resolved at runtime.
 
+- `chroot_namespace` `(string: "admin/")` – Specifies the namespace that will be used
```

Comment:
```suggestion
- `chroot_namespace` `(string: "admin")` – Specifies the namespace that will be used
```
We don't require the trailing slash, and it's probably more natural to the user to omit it.